### ISSUE TITLE
Question about modification

### ISSUE THREAD

**Issue by KORESHon** [Oarcinae/FactorioScenarioMultiplayerSpawn#260]
Is there any way to make the mod stand only on the server, and clients (players) do not need to download this mod?

**Comment by Oarcinae** [comment]
No, if I want to add proper mod compatibility and mod settings, it needs to be an actual mod, not a scenario. I won't be maintaining a scenario version anymore, sorry! There are too many benefits of having a mod instead of a scenario (updating, mod compatibility and mod settings are the biggest reasons).

You are welcome to take my mod and strip out all the mod parts to make it "just a scenario" but it might be quite difficult to do that.

**Comment by KORESHon** [comment]
Too bad, it's just the only mod on the server (not counting DLC). But thanks for such a quick response!

**Comment by Oarcinae** [comment]
> Too bad, it's just the only mod on the server (not counting DLC). But thanks for such a quick response!

I understand, I used to only provide a scenario for that reason (so there would be no mod to download).

If I have time after I finish adding support for space age, I will see how difficult it is to make a separate "pure scenario" version.
